lib/active_admin_import/model.rb

```diff
@@ -138,8 +138,10 @@ def define_methods_for(attr_name)
 
     def encode(data)
       data = content_encode(data) if force_encoding?
-      data = data.encode('UTF-8',
-                         invalid: :replace, undef: :replace)
+      data = data.encode(
+        'UTF-8',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.